NLP/BI-LSTM_CRF_Tensorflow/dataloader.py

```diff
@@ -0,0 +1,11 @@
+import pandas as pd
+
+# Downloading the dataset
+#!kaggle datasets download -d abhinavwalia95/entity-annotated-corpus
+# unzip the dataset and keep it in a folder named entity-annotaed-corpus
```

Comment:
You should add these kinds of instructions in `README.md` file along with commands that should be used to do achieve these. It is recommended to add those command in your `dataloader.py` file which automatically downloads data and use that - in that case no need to add commands in `README.md` - but this will also work for now after adding instructions in the readme file.